lib/rpcs/v1_4_5/get_account.rs

```diff
@@ -1,22 +1,39 @@
+use schemars::JsonSchema;
```

Comment:
This shouldn't be required.  We could look to remove it from any other types we apply it to in a follow-up PR.